config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -37,7 +37,7 @@ deployment_versions="
 "
 
 # The experimental version for which jobs are optional.
-experimental_k8s_version="1.26"
+experimental_k8s_version="1.28"
 
 # The latest stable Kubernetes version for testing alpha jobs
 latest_stable_k8s_version="1.25" # TODO: bump to 1.26 after testing a pull job
```

Comment:
Can you bump this to 1.26 as we've removed 1.25?  Then submit another PR to bump it to 1.28 after testing a pull job after the PR is merged.

Reply:
okay~